src/_components/NavBarComponent.jsx

```diff
@@ -1,35 +1,62 @@
-import React, { Component } from 'react';
-import jwt from 'jsonwebtoken';
-import { Dropdown, Menu, Icon } from 'semantic-ui-react';
-
-import '../_css/NavBarComponent.css';
-
-const NavBarComponent = (props) => {
-  const { title } = props;
-  const token = jwt.decode(localStorage.getItem('token'));
-  let username = '';
-
-  if (token != null) {
-    username = token.username;
-  }
-
-  return (
-    <div className='navbar'>
-      <Menu secondary stackable>
-        <Menu.Item name='menu' onClick={props.toggleVisibility}>
-          <Icon name='list layout'/>
-        </Menu.Item>
-        <Menu.Item name={title}/>
-        <Menu.Menu position='right'>
-          <Dropdown item text={username} icon='user' simple>
-            <Dropdown.Menu>
-              <Dropdown.Item>Logout</Dropdown.Item>
-            </Dropdown.Menu>
-          </Dropdown>
-        </Menu.Menu>
-      </Menu>
-    </div>
-  );
-};
-
-export default NavBarComponent;
+import React, { Component } from 'react';
+import jwt from 'jsonwebtoken';
+import { withRouter } from 'react-router-dom';
+import { Dropdown, Menu, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import '../_css/NavBarComponent.css';
+
+/* eslint require-jsdoc: 0 */

```

Comment:
Please get rid of all this comment. We won't need once we fix the JSLint configuration